index.js

```diff
@@ -3,6 +3,40 @@ const booksList = document.getElementById('books-list');
 const form = document.getElementById('form');
 const title = document.getElementById('title');
 const author = document.getElementById('author');
+const time = document.getElementById('time');
+const list = document.getElementById('list');
+const formSection = document.getElementById('form-section');
+const contact = document.getElementById('contact');
+
+// hide form and contact section
+formSection.style.display = 'none';
+contact.style.display = 'none';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] Please kindly do not disable linters, you should fix your linters instead of disabling them. Fix linters help us to write a clean and professional codebase. Kindly fix that 🙃 😉

Reply:
Thanks, @cynthiainga
Our project description allowed us to disable linters for unused variables, and that's why we did that.

I have refactored the code now though, and we haven't disabled linters anymore.
Hopefully, we get approved now 😊